diversity-inclusion.md

```diff
@@ -0,0 +1,30 @@
+# Statement on Diversity and Inclusion at Tech Events
+
+August 2021
+
+Diversity and inclusion in the technology industry are subjects that are close to the heart of the Open Web Docs team. We value fostering a diverse and inclusive technology industry because it is the right thing to do. If that isn’t a strong enough argument, having a diverse team and diverse representatives has proven again and again to create better products, foster a better work environment and, if you need the business case, to improve the bottom line.
+
+Tech events are a highly visible face of the tech industry. Not only are the speakers representatives of their respective companies and technologies, but they will act as role models for attendees and external audiences. It is therefore vital that speaker lineups represent the diverse range of people who work in technology and that events foster an inclusive and welcoming experience for all of their potential attendees, no matter their gender, ethnicity, sexual orientation, religion, national origin, disability or any other factor. Any organization that participates in tech events needs to pay attention to these issues and work to make these events more diverse and more welcoming.
+
+That is why we have signed up to support the [Diversity 
+Charter](https://diversitycharter.org) for technology events.
+
+!["Diversity Charter logo"](diversitycharter-horizontal.png)
+
+As part of our commitment, we will support diversity & inclusion in the following ways when it comes to events:
+
+- Our team will not take part in panel discussions of two or more people unless there is at least one person of an underrepresented group on the panel, not including the chair.
+
+- We will not speak at or sponsor an event unless the event’s organizers are clearly working hard to address the diversity of their speakers and attendees. We expect representation from different genders, ethnicities, abilities and other underrepresented groups in tech.
+
+- We will not participate in or sponsor events unless they operate under, and enforce, a published code of conduct. We also developed [our own code of conduct](code_of_conduct.md) adapted from the Contributor Covenant. We like codes of conduct that are specific and provide detailed enforcement information.
```

Comment:
I think this should be adapted... will think of something.

Reply:
Specifically this is a contribution code of conduct rather than event code of conduct. We might want to, in addition to mentioning our code of conduct, point to https://viewsourceconf.org/code-of-conduct/ as a good example of an event code of conduct.

Reply:
Pointing to the ViewSource CoC now. Does this look better?